lib/log_dcs.c

```diff
@@ -166,6 +172,17 @@ qb_log_dcs_get(int32_t * newly_created,
 	return cs;
 }
 
+struct qb_log_callsite *
+qb_log_dcs_get(int32_t * newly_created,
+	       const char *function,
+	       const char *filename,
+	       const char *format,
+	       uint8_t priority, uint32_t lineno, uint32_t tags)
+{
+	return qb_log_dcs_get2(newly_created, NULL, function,
+				filename, format, priority, lineno, tags);
+}
```

Comment:
qb_log_dcs_get() is an internal function and was only used in the one spot, so we can just drop it now (or use its name for the new function)

Reply:
Indeed. I have added the `message-id` to the `qb_log_dcs_get` and removed the `qb_log_dcs_get2`.